nuget/MacMobileCenter.nuspec

```diff
@@ -16,6 +16,11 @@
         <group targetFramework="netstandard1.0">
             <dependency id="NETStandard.Library" version="1.6.1" />
         </group>
+        <!-- Use dependency groups for iOS and Android so that NETStandard dependencies aren't added to them -->
```

Comment:
update comment so that it's not .net standard specific and so we can merge even with the rollback